onnxruntime/python/tools/pytorch_export_contrib_ops.py

```diff
@@ -98,8 +98,9 @@ def unregister():
         try:
             torch.onnx.unregister_custom_op_symbolic(name, _OPSET_VERSION)
         except AttributeError:
+            from torch.onnx import symbolic_registry
```

Comment:
Just out of curiosity, why do you import `symbolic_registry` here? It was imported right below `from torch.onnx import symbolic_helper`.

Reply:
The symbolic_registry module was removed in the latest pytorch; we are importing here for backwards compatibility. Added a comment.